### ISSUE TITLE
Contracts: category summary

### ISSUE THREAD

**Issue by heldergg** [jorgecarleitao/public-contracts#76]
The category summary I've mentioned on my wish (#75) could be enriched. For instance:

* Statistics about value (most expensive contract, average contract, etc);
* Statistics about contracted entities (most often contracted company, company with the greatest money volume of contracts, etc).

**Comment by jorgecarleitao** [comment]
I agree. Categories didn't received much love since I introduced them. :X
